generate.go

```diff
@@ -16,6 +17,22 @@ func main() {
 		CompileProtos:      true,
 		SkipGenMocks:       true,
 		SkipGeneratedTests: true,
+		ProtoDepConfig: &protodep.Config{
+			Local: &protodep.Local{
+				Patterns: []string{"test/**/*.proto", "api/**/*.proto", protodep.SoloKitMatchPattern},
+			},
+			Imports: []*protodep.Import{
+				{
+					ImportType: &protodep.Import_GoMod{GoMod: protodep.ExtProtoMatcher},
+				},
+				{
+					ImportType: &protodep.Import_GoMod{GoMod: protodep.ValidateProtoMatcher},
+				},
+				{
+					ImportType: &protodep.Import_GoMod{GoMod: protodep.GogoProtoMatcher},
+				},
+			},
+		},
```

Comment:
i am concerned with the complexity of this new config for solo-kit generate. unlike the existing options which are pretty simple / easy to understand for a basic solo-kit user, the options provided here require an in-depth understanding of the new `protodep` pacakge.

I think for the usability of Solo-Kit we need to simplify this process for our solo-kit projects, so we do not wind up with magical/overcomplex `generate.go` in all of our projects

Reply:
I was thinking this as well, I can abstract away some of the basic calls. For instance if you look at the Gloo PR, it is much simpler. There will most likely have to be user additions of some kind no matter what though.